File: DesignPatternsExercises/java/MementoDesignPattern/Exercise/Memento.java
public class Memento {
    private final LedTV ledTV;
    private final Speaker speaker;

    public Memento(LedTV ledTV, Speaker speaker) {
        this.ledTV = ledTV;
        this.speaker = speaker;
    }

    public LedTV getLedTV() {
        return ledTV;
    }

    public Speaker getSpeaker() {
        return speaker;
    }
}

File: DesignPatternsExercises/ruby/memento_design_pattern/solution/memento.rb
class Memento
  # attr_reader __________(7)__________, :speaker
  attr_reader :led_tv, :speaker

  # def __________(8)__________(led_tv, speaker)
  def initialize(led_tv, speaker)
    @led_tv = led_tv
    @speaker = speaker
  end
end
